sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/CalciteQueryPlanner.java

```diff
@@ -75,7 +75,9 @@
   private final Planner planner;
   private final JdbcConnection connection;
 
-  public CalciteQueryPlanner(JdbcConnection connection, RuleSet[] ruleSets) {
+  public CalciteQueryPlanner(
+      JdbcConnection connection, RuleSet[] ruleSets, String defaultTimezone) {
+    // TODO: support setting default timezone?
```

Comment:
Can you create and link a JIRA?

Reply:
Actually this config does not seem to apply to CalciteSQL, so I have reverted this change.

Reply:
> Actually this config does not seem to apply to CalciteSQL

Why not?

Reply:
Default timezone affects the result of many time-related functions in ZetaSQL. In CalciteSQL language spec, I did not find any mention of default timezone. I guess this concept may not apply to CalciteSQL, or it does not affect CalciteSQL in the same way. So I am leaving the CalciteSQL part unchanged and make it clear the new option only applies to ZetaSQL dialect.